openapi/demand-surge-api.yaml

```
@@ -0,0 +1,199 @@
+openapi: 3.1.0
+info:
+  title: PredictHQ Demand Surge API
+  description: Identify dates with surges in demand caused by multiple events happening at the same time and location.
+  version: 1.0.0
+  license:
+    name: MIT
+    url: 'https://opensource.org/licenses/MIT'
+  termsOfService: 'https://www.predicthq.com/legal/terms'
+security:
+  - BearerAuth: []
+tags:
+  - name: Demand Surge API
+    externalDocs:
+      url: 'https://docs.predicthq.com/api/demand-surge/get-demand-surges'
+      description: Demand Surge Docs
+servers:
+  - url: 'https://api.predicthq.com'
+externalDocs:
+  url: 'https://docs.predicthq.com'
+  description: PredictHQ Tech Docs
+x-hideTryItPanel: true
+paths:
+  /v1/demand-surge/:
+    get:
+      operationId: get_demand_surge_demand_surge_get
+      summary: Get Demand Surges
+      parameters:
+        - name: min_surge_intensity
+          in: query
+          description: |-
+            Filters out demand surges smaller than the minimum surge intensity provided.
+
+
+            E.g. `?min_surge_intensity=m`
+          required: true
+          schema:
+            $ref: '#/components/schemas/SizeEnum'
+        - name: date_from
+          in: query
+          description: |-
+            The beginning of the demand surge search window. The demand surge will be looked up over the next 90 days after the date you provide for the `date_from` parameter.
+
+
+            The accepted format for this parameter is `YYYY-MM-DD`
+
+
+            E.g. `?date_from=2021-05-12`
+          required: true
+          schema:
+            type: string
+            format: date
+            title: Date From
+        - name: location.place_id
+          in: query
+          description: |-
+            A comma-separated list in the format representing the place ids of a location in the format `<place_id1>,<place_id2>` . It cannot be used with `origin` or `radius` suffixes.
+
+
+            E.g. `?location.place_id=2643743,2643744`
+          required: false
+          schema:
+            description: A comma-separated list of place ids
+            type: array
+            items:
+              type: string
+              examples:
+                - 'value1,value2'
+            title: Location.Place Id
+          explode: false
+          style: form
+        - name: location.origin
+          in: query
+          description: |-
+            A comma-separated coordinate (`<lat>,<lon>`) representing the centroid of a location. It must be used with `radius` suffix and cannot be used with `location.place_id`
```

```suggestion
            A comma-separated coordinate (`<lat>,<lon>`) representing the centroid of a location. It must be used with `location.radius` and cannot be used with `location.place_id`
```

gah sorry i missed this last time